<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sway analysis toolkit (SwAT)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>gibbon.jpg</normaloff>gibbon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filtering</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QComboBox" name="orderCombo">
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cuttoffCombo">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>27</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="detrend">
        <property name="text">
         <string>Detrend</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>20</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>No filter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Butterworth filter</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="minSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="minSpinBoxSuffix">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sliderField"/>
      </item>
      <item>
       <widget class="QSpinBox" name="maxSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="maxSpinBoxSuffix">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resultantLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resultantSpinBoxSuffix">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="rawDataViewerLabel">
        <property name="text">
         <string>Raw data viewer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="orderLabel">
        <property name="text">
         <string>Order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cuttoffLabel">
        <property name="text">
         <string>Cuttoff (Hz)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser2"/>
      </item>
      <item>
       <widget class="QLabel" name="timeAdjustLabel">
        <property name="text">
         <string>Time adjust</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Ctrl_O"/>
    <addaction name="actionClose_Ctrl_Q"/>
   </widget>
   <widget class="QMenu" name="menuAnalyse">
    <property name="title">
     <string>Analyse</string>
    </property>
    <addaction name="actionBasic"/>
   </widget>
   <widget class="QMenu" name="menuPlotting">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionPlot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalyse"/>
   <addaction name="menuPlotting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Ctrl_O">
   <property name="text">
    <string>Open   </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_Ctrl_Q">
   <property name="text">
    <string>Close   </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBasic">
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot builder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
